atoMEC/writeoutput.py

```diff
@@ -524,17 +524,15 @@ def density_to_csv(rgrid, density, filename):
         name of the file to write to
     """
     if config.spindims == 2:
-        headstr = (
-            "r (a_0)"
-            + 4 * " "
-            + "n^u (orbs)"
-            + 3 * " "
-            + "n^u (ideal)"
-            + 2 * " "
-            + "n^d (orbs)"
-            + 3 * " "
-            + "n^d (ideal)"
-            + 1 * " "
+        headstr = " ".join(
+            s.rjust(12)
+            for s in [
+                "r (a_0)",
+                "n^u (orbs)",
+                "n^u (ideal)",
+                "n^d (orbs)",
+                "n^d (ideal)",
+            ]
         )
```

Comment:
This looks much nicer. I think a comment a la "# right-justifing column heads with a fixed with of 12 characters" would make things even clearer. For now, lets stick with your improved code. Maybe we should add a new issue that explores ways to improve saving of the output, for example by using the `csv` module.

Reply:
Added comment and issue #125 so we don't forget